compiler/luci/pass/src/FuseBatchNormWithConvPass.cpp

```diff
@@ -200,6 +201,8 @@ bool fused_batch_norm_with_conv(luci::CircleAdd *add)
   fused_conv->dilation()->h(conv->dilation()->h());
   fused_conv->dilation()->w(conv->dilation()->w());
   fused_conv->name(name + "/Conv2D");
+  luci::add_origin(fused_conv, luci::composite_origin({luci::get_origin(add), luci::get_origin(mul),
+                                                       luci::get_origin(conv)}));
```

Comment:
Q) just a question, is this equivalent? for understanding...
```cxx
luci::add_origin(fused_conv, luci::get_origin(add));
luci::add_origin(fused_conv, luci::get_origin(mul));
luci::add_origin(fused_conv, luci::get_origin(conv));
```

Reply:
@seanshpark Yes:)